omnigibson/prims/xform_prim.py

```diff
@@ -145,6 +133,7 @@ def _set_xform_properties(self):
             xform_op_rot = lazy.pxr.UsdGeom.XformOp(self._prim.GetAttribute("xformOp:orient"))
         xformable.SetXformOpOrder([xform_op_translate, xform_op_rot, xform_op_scale])
 
+        # TODO: This is the line that causes Transformation Change on... errors. Fix it.
```

Comment:
did we ever figure out why this error sometimes occurs? so far IIRC the biggest cause was having negative scale assigned to objects / prims

Reply:
TODO: @cgokmen

Reply:
Just to be clear this is talking about a lot of "transform change not allowed on child link of articulation" warnings shown on the log e.g. it's about a benign but very noisy issue. @hang-yin please open a ticket, P2, to fix this.